computeLocalExtremaMetrics.py

```diff
@@ -144,41 +144,26 @@ def computeLocalExtremaMetrics(varConStenDex, varCon, varCoord, varS2T, varST, a
        
        # Compute normalization integrals
        L1Den = computeGlobalWeightedIntegral(NT, varCon, np.abs(varST), areaT, jacobiansT, SpectralElement)
-       L2Den = computeGlobalWeightedIntegral(NT, varCon, np.abs(varST2), areaT, jacobiansT, SpectralElement)
+       L2Den = computeGlobalWeightedIntegral(NT, varCon, varST2, areaT, jacobiansT, SpectralElement)
        LinfDen = np.amax(varST) - np.amin(varST)
+       if LinfDen < 1E-14: LinfDen = 1.0 # max and min values are same
        
        # Compute numerators for minima
-       varDiff = minDiff
-       varDiff2 = np.power(minDiff, 2)
-       L1Num = computeGlobalWeightedIntegral(NT, varCon, varDiff, areaT, jacobiansT, SpectralElement)
-       L2Num = computeGlobalWeightedIntegral(NT, varCon, varDiff2, areaT, jacobiansT, SpectralElement)
-       LinfNum = np.amax(varDiff)
+       L1Num = computeGlobalWeightedIntegral(NT, varCon, minDiff, areaT, jacobiansT, SpectralElement)
+       L2Num = computeGlobalWeightedIntegral(NT, varCon, np.power(minDiff, 2), areaT, jacobiansT, SpectralElement)
+       LinfNum = np.amax(minDiff)
 
-       LareaDen = computeGlobalWeightedIntegral(NT, varCon, np.ones(varST.shape), areaT, jacobiansT, SpectralElement)
-       
-       if SpectralElement:
-              Lmin_1 = np.asscalar(L1Num / L1Den)
-              Lmin_2 = mt.sqrt(L2Num / L2Den)
-              Lmin_inf = LinfNum / LinfDen
-       else:
-              Lmin_1 = np.asscalar(L1Num / NT / LinfDen / LareaDen)
-              Lmin_2 = mt.sqrt(L2Num / NT) / LinfDen / LareaDen
-              Lmin_inf = LinfNum / LinfDen
+       Lmin_1 = np.asscalar(L1Num / L1Den)
```

Comment:
Not very important, but asscalar is deprecated, so could try using (L1Num / L1Den).item()

Reply:
Will do.